index.html

```diff
@@ -0,0 +1,186 @@
+            </path>
+          </g>
+        </svg>
+      </div>
+      <div class="d-flex justify-content-between align-items-center ">
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <a href="#" class="d-none d-lg-inline-block">LOGIN</a>
+        <button class="btn btn-warning roboto-font rounded-pill">
+          SUBSCRIBE &rsaquo;
+        </button>
```

Comment:
- [ ] The fonts and colors of these are also not the same as the original, please try achieving a result as close to the original as possible :smile: